Gemfile

```diff
@@ -1,5 +1,5 @@
-ruby "3.3.0"
-source "https://ezcater.jfrog.io/ezcater/api/gems/ezcater-gem-source"
+ruby "3.3.3"
+source "https://rubygems.org"
```

Comment:
Updating this source as none of the gems used in this repo needed to use our private gem source.